modules/highgui/src/window_w32.cpp

```diff
@@ -596,6 +596,94 @@ void cvSetPropTopmost_W32(const char* name, const bool topmost)
     }
 }
 
+double cvGetPropVsync_W32(const char* name)
+{
+#ifndef HAVE_OPENGL
+    CV_UNUSED(name);
+    CV_Error(Error::OpenGlNotSupported, "Library was built without OpenGL support");
+    return -1;
+#else
+    CV_Assert(name);
+
+    if (!name)
+        CV_Error(Error::StsNullPtr, "NULL name string");
```

Comment:
The condition duplicates previous `CV_Accert`.

Reply:
absolutely! I'll keep the CV_Error and remove the assertion.